q2-structure/proposal-by-xdg.md

```diff
@@ -0,0 +1,219 @@
+the ongoing development of the Perl language and interpreter.  They do not
+need to be language developers or committers.
+
+### Powers
+
+The Core Team has authority over the rules and composition of Perl governance.
+
+They alone elect the Steering Council and have the power to remove Steering
+Council members.
+
+In concert with the Steering Council, they manage their own membership and
+have the power to modify the Perl Rules of Governance.
+
+The Core Team do not have any authority over parts of the Perl ecosystem
+unrelated to developing and releasing the language itself.
+
+### Membership, Inactivity, and Removal
+
+The initial Core Team members will be specified in the first draft of the Perl
+Rules of Governance.
+
+Core Team members who have stopped contributing are encouraged to declare
+themselves as "inactive". Inactive members do not participate in votes.
+Inactive members may declare themselves active at any time, except when a vote
+has been proposed and is not concluded.  Eligibility to vote will be
+determined by the Vote Administrator.
+
+Core Team Members may also resign their position at any time.
+
+Any Core Team member may nominate someone to be added to the Core Team by
+sending the nomination to the Steering Council.  The Steering Council must
+approve or reject the nomination.  If approved, the Steering Council will
+organize a Membership Change Vote to ratify the addition.
+
+In exceptional circumstances, it may be necessary to remove someone from the
+Core Team against their will, such as for flagrant or repeated violations of a
+Code of Conduct.  Any Core Team member may send a recall request to the
+Steering Council naming the individual to be removed.  The Steering Council
+must approve or reject the recall request.  If approved, the Steering Council
+will organize a Membership Change vote to ratify the removal.
+
+If the removed member is also on the Steering Council, then they are removed
+from the Steering Council as well.
+
+### No Confidence in the Steering Council
+
+The Core Team may remove either a single Steering Council member or the entire
+Steering Council via a No Confidence Vote.
+
+A No Confidence Vote is triggered when a Core Team member calls for one
+publicly on an appropriate project communication channel, and another Core
+Team member seconds the proposal.
+
+If a No Confidence Vote removes all Steering Council members, the Vote
+Administrator of the No Confidence Vote will then administer a Term Election
+to select a new Steering Council.
+
+### Amending Perl Rules of Governance
+
+Any Core Team member may propose amending the Perl Rules of Governance by
+sending a proposal to the Steering Council.  The Steering Council must decide
+to approve or reject the proposal.  If approved, the Steering Council will
+administer an Amendment Vote.
+
+## The Perl Steering Council
+
+The Perl Steering Council is a 5-person committee, elected by the Perl Core
+Team.
```

Comment:
I think we'll do better with three than five.  I know we had a straw poll about this, but we didn't have much back and forth, because the ballot was on 1 vs. >1.

I believe the argument for 5 was mostly "it's awkward to be the deciding vote between your two coworkers," but 5 doesn't eliminate that.  Maybe it makes it more rare, and maybe it doesn't, but it's there.  On the other hand, I suspect that in *practice* a lot of the responsibilities of the council will be delegated to individual members.  "Pat, you handle X and I'll handle Y."  If that holds, then I think it'll be much easier for 3 people to come to a consensus on that front than for 5.

If we get no other proposal of note, I'm tempted to suggest "exactly this proposal, except 3 instead of 5" as an alternative, and then be happy with either result.

Reply:
The argument against three that resonated most for me was that it sucks to be on the losing end of a 2-1 vote, particularly if that happens a lot -- easy to feel ganged up on.  Losing 3-2 means at least one person agreed with you.  And losing 4-1 isn't close like 2-1 so there's more acceptance of strong views for going a different way.  So I see 5 as optimizing for the experience of the minority on the council, maybe leading to richer discussion/debate vs a solitary dissenter deciding to check out.

But I have no objection to you running a "this proposal, but 3" alternative.